### ISSUE TITLE
issue with tht schematic

### ISSUE THREAD

**Issue by heneault** [bvenneker/Chat64#8]
I think there are some issues in the latest schematic of the THT version:

- 74hct125 should not be run at 3.3v. According to spec, vcc should be between 4.5 V to 5.5 V For HCT circuits operating at less than 4.5 V, the load-level noise margin is reduced and becomes incompatible with TTL thresholds.

- NMI line is open collector. if NMI if pulled on commodore side (i.e. pressing restore key) a short circuit will on pin 6 of U8 of the C64.

- there is no need to connect oNMI to 74hct125 as this is already 3v

- the 74hc597 has a min Vih of 3.15 v when vcc is at 4.5v.  Since you are running at 5v it is a little be higher and probably close to 3.3v . That could affect high logic detection of pload and sclk.

**Comment by bvenneker** [comment]
Hi,
Thanks for your input again!

About the nmi line.
So if the restore key is pressed, U8 (in the C64) tries to pull that signal
low while the cartridge tries to hold it high. So we can fix that by also
using an open collector on our side, thus using an npn transistor without
going through the 74hc125, that would also fix your third remark. Agree??

Best,
Bart


Op do 4 apr 2024 om 03:01 schreef Yannick Heneault ***@***.***

**Comment by heneault** [comment]
yes that would fix it. But I think you must also add a resistance between the collector and the cartridge pin as the voltage of the collector at saturation is near 0v (but not 0v). The resistance will limit the current that will sink on the line. Make sure the current is limited to what the ttl chip in the c64 can handle.

Otherwise, you can use something like a 74ls06
